### ISSUE TITLE
UnusedPrivateElementsSniff.UnusedMethod - false positive with Closure::fromCallable()

### ISSUE THREAD

**Issue by enumag** [slevomat/coding-standard#219]
Closure::fromCallable is very useful when you want to pass a private method as callback. The sniff should recognize such method as used.

```php
class Foo
{
    public function test()
    {
        array_map(\Closure::fromCallable([$this, 'map']), []);
    }

    private function map($item)
    {
        // ...
    }
}
```

**Comment by kukulich** [comment]
If you prepare PR, I'm willing to merge it. However I don't want to invest my time for supporting the old way `[$this, 'map']`.

You should really rewrite the code in the example to:
```
array_map(function ($item) {
	return $this->map($item);
}, []);
```

**Comment by enumag** [comment]
That's the syntax I used before. It's too much writing and forces you to repeat all the parameters. No thanks.

**Comment by TomasVotruba** [comment]
@enumag Yet it's cleaner for most IDEs and static analysis tools.
